src/App.tsx

```diff
@@ -1,24 +1,73 @@
 import React from 'react';
 import './App.scss';
+import Clock from './components/Clock';
 
-const App: React.FC = () => {
-  setInterval(() => {
-    const date = new Date();
-
-    // eslint-disable-next-line
-    console.log(date.toLocaleTimeString());
-  }, 1000);
-
-  return (
-    <div className="App">
-      <h1>React clock</h1>
-      <p>
-        Current time:
-        {' '}
-        {/* Print the time here instead of DevTools */}
-      </p>
-    </div>
-  );
+type State = {
+  isClockVisible: boolean,
+  name: string,
 };
 
+class App extends React.Component<{}, State> {
+  state: State = {
+    isClockVisible: true,
+    name: 'React clock',
+  };
+
+  ShowClock = () => {
```

Comment:
I think that class methods should start from small letter
```suggestion
  showClock = () => {
```

Reply:
fixed